api/src/LinkyLink/Controllers/OpenGraphController.cs

```diff
@@ -32,31 +29,21 @@ public ValidatePageController(IOpenGraphService openGraphService)
         /// <remarks>
         /// Sample body:
         ///
-        ///     {
+        ///     [{
         ///        "id": 1,
-        ///        "url": "https://www.google.com"
-        ///     }
+        ///        "url": "https://www.microsoft.com"
+        ///     }]
         ///
         /// </remarks>        
-        // POST: api/ValidatePage
+        // POST: api/OpenGraph
         [HttpPost]
-        public async Task<ActionResult<OpenGraphResult>> Post()
+        public async Task<ActionResult<OpenGraphResult>> Post(IEnumerable<OpenGraphRequest> openGraphRequests)
```

Comment:
Previously this end point supported both the array of objects and a object. I dont how the front end works but FE may need to be updated to uptake this change to always send an array in the body. Let Jafar know or create a new task to track that.

Reply:
Created new User Story to track this change.  Has detailed information on what was changed on BE and what change is needed on FE.
https://dev.azure.com/cseeest/OpenHack/_workitems/edit/1009